memory/memhotplug.py

```diff
@@ -38,6 +38,15 @@
             'double fault:', 'BUG: Bad page state in']
 
 
+def get_mem_count():
+    with open('/sys/devices/system/memory/block_size_bytes', 'r') as block_file:
+        blocksize = int(block_file.read()) / 10
+    with open('/sys/devices/system/node/has_normal_memory', 'r') as node_file:
+        nodes = node_file.read()
+    node_mem = (memory.meminfo.MemFree.mb) / len(nodes.split('-'))
```

Comment:
in my system i get `has_normal_memory ` value is `0,8`  so this split logic not works for me can you please more elaborate why it needed